request.proto

```diff
@@ -285,6 +286,14 @@ message EquipmentReportsRequest {
     repeated string forbidden_uris  = 5;
 }
 
+message AccessPointsRequest {
+    optional int32 depth            = 1;
+    optional int32 start_page       = 2;
+    optional int32 count            = 3;
+    optional string filter          = 4;
```

Comment:
It is a bit hard to understand the meaning of the `filter` and `forbidden_uri` fields.
Could you add a comment explaining what these should contain ?

Reply:
Sorry, but this is for all the APIs it is like that... 
This is true, when you don't know the code it is just the foggy code